Tests/SwiftProtobufTests/Test_JSONEncodingOptions.swift

```diff
@@ -17,9 +17,23 @@ import XCTest
 import SwiftProtobuf
 
 class Test_JSONEncodingOptions: XCTestCase {
+  
+  func testAlwaysPrintInt64sAsNumbers() {
+    // Use explicit options (the default is false), no reason only others can be pedantic.
+    var asStrings = JSONEncodingOptions()
+    asStrings.alwaysPrintInt64sAsNumbers = false
+    var asNumbers = JSONEncodingOptions()
+    asNumbers.alwaysPrintInt64sAsNumbers = true
+
+    let msg1 = ProtobufUnittest_Message2.with {
+      $0.optionalInt64 = 1656338459803
+    }
+    XCTAssertEqual(try msg1.jsonString(options: asStrings), "{\"optionalInt64\":\"1656338459803\"}")
+    XCTAssertEqual(try msg1.jsonString(options: asNumbers), "{\"optionalInt64\":1656338459803}")
+  }
 
```

Comment:
We should also have a test here to verify that `map<int64,string>` still correctly writes the int64 in quoted form even with this option set.  Ideally, we'd also check `map<uint64,string>` for completeness.